Resources/Prototypes/Objectives/stealTargetGroups.yml

```diff
@@ -84,6 +84,13 @@
     sprite: Objects/Weapons/Guns/Battery/energy_shotgun.rsi
     state: base
 
+- type: stealTargetGroup
+  id: OrwellianCircuitBoard
+  name: steal-target-groups-orwellian-circuitboard #Here at the syndicate we love to control people. Do you love to control people? Because we do.
```

Comment:
> Please remove the comment with the string. It's not needed.
> 
> I also don't see `steal-target-groups-orwellian-circuitboard` declared anywhere. Did you test this? It will show up as literal `steal-target-groups-orwellian-circuitboard`

Removed the comment! May I ask where I can declare the "steal-target-groups-orwellian-circuitboard"?

Reply:
Upstream defines `steal-target-groups` loc strings in `Resources/Prototypes/Objectives/stealTargetGroups.yml`

So that would be `Resources/Prototypes/_Harmony/Objectives/stealTargetGroups.yml`

Reply:
Brainfart. That's yaml for entities.

`Resources/Locale/en-US/objectives/conditions/steal-target-groups.ftl` and by extension `Resources/Locale/en-US/_Harmony/objectives/conditions/steal-target-groups.ftl` is the FTL.

Reply:
Resolved as the steal target has been removed.